### DISCUSSION TITLE
How to set the reply in comment widget to be expaned by default.

### DISCUSSION THREAD

**Discussion by Jianbin719** [YetiForceCompany/YetiForceCRM#16425] [How to]
How to set the reply in comment widget to be expaned by default. 
Original: 
![image](https://user-images.githubusercontent.com/75652286/187577152-a37515b1-dc7b-4e1c-bf09-d27da5c5877f.png)

Expected: expand reply content by default
![image](https://user-images.githubusercontent.com/75652286/187577331-2bcdb23f-4520-4936-b390-96bf0f65f8b9.png)

**Comment by vovpff** [comment]
I'm interested too

**Reply by Akaki-1991** [reply]
How can I set up a workflow so that whoever I return a comment to receives an email notification

**Reply by Jianbin719** [reply]
Worklfow can do it. I think.

**Comment by Jianbin719** [comment]
Anyone know, the reply to folded is very not convenient. sometimes can not be found?? somewhere we just modify the js??

**Comment by Jianbin719** [comment]
<script>
window.onload = function() {
    var buttons = document.querySelectorAll('.viewThread');
    buttons.forEach(function(button) {
        button.click(); 
    });
};

</script>

**Comment by Jianbin719** [comment]
I made it by adding this code to the end of filed in /layouts/basic/modules/Vtiger/Comment.tpl

`......

			
<script>
window.onload = function() {
    var buttons = document.querySelectorAll('.viewThread');
    buttons.forEach(function(button) {
        button.click(); 
    });
};

</script>

{/strip}`
